### PR TITLE
[Docs] Broaden description of managed infra

### PR DESCRIPTION
**Pull request by hamza-mohd**
**Notes for Reviewers**

This PR broadens the overview description of the Meshery project.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — DISMISSED** [review]
Good. DCO is in place, too.

File: docs/pages/getting-started/overview.md [hunk 1]
```diff
@@ -30,37 +32,33 @@ Meshery features can be categorized by:
...
 
-Beyond the need for performance and overhead data under a permutation of different workloads (applications) and types and sizes of infrastructure resources, the need for cross-project, apple-to-apple comparisons are also desired in order to facilitate a comparison of behavioral differences between service meshes and selection of their use. Individual projects shy from publishing test results of other, competing service meshes. An independent, unbiased, credible analysis is needed.
+Beyond the need for performance and overhead data under a permutation of different workloads (applications) and types and sizes of infrastructure resources, the need for cross-project, apple-to-apple comparisons are also desired in order to facilitate a comparison of behavioral differences between cloud antive  and selection of their use. Individual projects shy from publishing test results of other, competing service meshes. An independent, unbiased, credible analysis is needed.
```
Comment (leecalcote) [inline docs/pages/getting-started/overview.md:45]: Misspelling here.

Comment (jamieplu) [inline docs/pages/getting-started/overview.md:45]:
```suggestion
Beyond the need for performance and overhead data under a permutation of different workloads (applications) and types and sizes of infrastructure resources, the need for cross-project, apple-to-apple comparisons are also desired in order to facilitate a comparison of behavioral differences between cloud native and selection of their use. Individual projects shy from publishing test results of other, competing service meshes. An independent, unbiased, credible analysis is needed.
```

File: docs/pages/getting-started/overview.md [hunk 2]
```diff
@@ -30,37 +32,33 @@ Meshery features can be categorized by:
 
 Whether making a Day 0 adoption choice or maintaining a Day 2 deployment, Meshery has useful capabilities in either circumstance. Targeted audience for Meshery project would be any technology operators that leverage service mesh in their ecosystem; this includes developers, devops engineers, decision makers, architects, and organizations that rely on microservices platform.
 
-### Meshery is for service mesh patterns
+### Meshery is for cloud native patterns
 
-Meshery integrates with Open Application Model (OAM) to enable users to deploy service mesh patterns.
+Through MeshModel, Meshery describes infrastructure under management, enabling you to define cloud native designs and patterns and then to export those designs and share within the <a href="https://meshery.io">Meshery Catalog</a>.
```
Comment (jamieplu) [inline docs/pages/getting-started/overview.md:37]: Thanks for the explanation of MeshModel, @leecalcote

**Comment by leecalcote** [comment]
Assuming build completes with deployment preview, we're good to go.

**Review by leecalcote — DISMISSED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 95e3f47800361cc3100a298b59ff083701bb0045 at: https://647e83db1c815a3aab9b3253--meshery-docs-preview.netlify.app

File: docs/pages/getting-started/overview.md [hunk 3]
```diff
@@ -5,14 +5,16 @@ permalink: /getting-started/overview
 language: en
 ---
 
-As the cloud native management plane, Meshery enables the adoption, operation, and management of any service mesh and their workloads. Meshery's powerful performance management functionality is accomplished through implementation of [Service Mesh Performance](https://smp-spec.io) (SMP). Meshery's cloud native management functionality leverages [Service Mesh Interface](https://smi-spec.io) (SMI) and Meshery is the conformance tool for SMI. Meshery integrates with Open Application Model (OAM) to enable users to deploy service mesh patterns. Meshery enables operators to deploy WebAssembly filters to Envoy-based data planes. Meshery facilitates learning about functionality and performance of service meshes and incorporates the collection and display of metrics from applications running on or across service meshes.
+As the cloud native management plane, Meshery enables the adoption, operation, and management of Kubernetes clusters and their lifecycle. Meshery's powerful performance management functionality is useful whether your running workloads on Kubernetes or outside of Kubernetes.
```
Comment (adithyaakrishna) [inline docs/pages/getting-started/overview.md:8]:
```suggestion
As the cloud native management plane, Meshery enables the adoption, operation, and management of Kubernetes clusters and their lifecycle. Meshery's powerful performance management functionality is useful whether you are running workloads on Kubernetes or outside of Kubernetes. 
```

File: docs/pages/getting-started/overview.md [hunk 4]
```diff
@@ -30,37 +32,33 @@ Meshery features can be categorized by:
...
 
-Beyond the need for performance and overhead data under a permutation of different workloads (applications) and types and sizes of infrastructure resources, the need for cross-project, apple-to-apple comparisons are also desired in order to facilitate a comparison of behavioral differences between service meshes and selection of their use. Individual projects shy from publishing test results of other, competing service meshes. An independent, unbiased, credible analysis is needed.
+Beyond the need for performance and overhead data under a permutation of different workloads (applications) and types and sizes of infrastructure resources, the need for cross-project, apple-to-apple comparisons are also desired in order to facilitate a comparison of behavioral differences between cloud native and selection of their use. Individual projects shy from publishing test results of other, competing service meshes. An independent, unbiased, credible analysis is needed.
```
Comment (adithyaakrishna) [inline docs/pages/getting-started/overview.md:45]: @leecalcote IMO the word service meshes makes more sense here or if cloud native is needed, can we rephrase the sentence? 

CC @jamieplu

**Comment by leecalcote** [comment]
fixes #7797

**Comment by l5io** [comment]
🚀 Preview for commit 3b7f915726bd4729992e7516dcbcba65ed36a036 at: https://647f2c9329b2f725baaa14e5--meshery-docs-preview.netlify.app

File: docs/pages/getting-started/overview.md (hunk 4 above)
Comment (leecalcote) [reply docs/pages/getting-started/overview.md:45]: @adithyaakrishna you're correct. 👍

**Review by leecalcote — APPROVED** [review]

**Pull request merged**